README.md

```diff
@@ -299,6 +300,28 @@ In order to rotate your log files, add in `/etc/logrotate.d/myapp`:
 }
 ```
 
+<a name="transports"></a>
+## What happened to my transports?
+
+There will never be an API for transports, or support for ObjectMode Writable streams.
+This library is fast because it does way less than the others. We went a
+great length to make sure this library is _really fast_, and transports
+will slow things down.
+
+So, how do you do a transport? You create another node process to do that, and you pipe them together.
+It's the Unix philosophy.
+
+Something like:
+
+```js
+var split = require('split2')
+var pump = require('pump')
+
+pump(process.stdin, split2(JSON.parse), myAwesomeTransport)
+```
+
+Doing it inside the same process will add unnecessary load to your single-threaded node.js, effectively slowing you down.
```

Comment:
Using transports in the same process causes unnecessary load and slows down Node's single threaded event loop.